core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
> we just log & ignore, and continue parsing the OSM data.

If it is bigger than the accepted range we could ignore but maybe it is safer to still include it.

If it is smaller we have to throw an exception or warn the user **and** include it as a restriction (even it is wider)?